README.md

```diff
@@ -2,101 +2,56 @@
 
 with _VERSION_ being the version. 
 The version matches the GraalVM version used in the image, for example: `21.1.0-java11`, `21.1.0-java16`...
 
 ```shell
 quay.io/quarkus/ubi-quarkus-native-s2i:21.1.0-java11 <-- GraalVM 21.1.0 with java 11 support
 quay.io/quarkus/ubi-quarkus-native-s2i:21.1.0-java16 <-- GraalVM 21.1.0 with java 16 support
-quay.io/quarkus/ubi-quarkus-native-binary-s2i:1.0 <-- Native binary s2i
+quay.io/quarkus/ubi-quarkus-native-binary-s2i:2.0 <-- Native binary s2i
 ```
 
 NOTE: You may wonder why we don't use `latest`. It's because `latest` has introduced more problems than benefits especially when reproducing issues.
 For this reason, we recommend using a stable version.
 
 ## Build Prerequisites
 
-### CEKit
-
-To build you need CEKit. We recommend using `virtualenv` to run `cekit`.
-On MacOS X and Linux, you can run the build as follows:
-
-```bash
-virtualenv --python=python3 .cekit
-source .cekit/bin/activate
-pip install -r requirements.txt
-
-# Run the scripts from the .github directory
-# i.e. .github/build-native-images.sh
-```
-
-For other Systems, please refer to the CEKit docs.
-
-### s2i
-
-s2i are needed for tests, download your OS version from https://github.com/openshift/source-to-image/releases/latest and put it in `s2i` folder. 
-
-Something like the following:
-
-```shell
-mkdir s2i
-cd s2i
-tar -zxvf source-to-image-*.tar.gz
-```
+* Apache Maven
+* Docker with Build Kit / `buildx` - See https://docs.docker.com/develop/develop-images/build_enhancements/.
 
 ## Build
 
-The build is controlled by 5 _image_ files:
-
-* `quarkus-native-image.yaml` produces [`ubi-quarkus-native-image`](https://quay.io/repository/quarkus/ubi-quarkus-native-image) images
-* `quarkus-mandrel.yaml` produces [`ubi-quarkus-mandrel`](https://quay.io/repository/quarkus/ubi-quarkus-mandrel) images
-* `quarkus-micro-image.yaml` produces the [`quarkus-micro-image`](https://quay.io/quarkus/quarkus-micro-image) image, a runtime image to execute your native executable
-* `quarkus-native-binary-s2i.yaml` produces the [`ubi-quarkus-native-binary-s2i`](https://quay.io/repository/quarkus/ubi-quarkus-native-binary-s2i) image
-* `quarkus-native-s2i.yaml` produces the [`ubi-quarkus-native-s2i`](https://quay.io/repository/quarkus/ubi-quarkus-native-s2i) images
-* `quarkus-tooling.yaml` produces the [`centos-quarkus-maven`](https://quay.io/repository/quarkus/ubi-quarkus-native-s2u) images (deprecated)
-
-To build the images, you must pass the "GraalVM" version as parameter (except for `quarkus-mandrel.yaml`, `quarkus-native-binary-s2i.yaml` and `quarkus-micro-image.yaml`):
-
 ```shell
-cekit --descriptor ${IMAGE} build \
-        --overrides "{'version': '${version}', 'modules': {'install': [{'name':'graalvm', 'version': '${version}'}]}}" \
-        docker --tag="${IMAGE_NAME}:${version}"
+> mvn install
 ```
 
-For `quarkus-mandrel.yaml` you must pass the "Mandrel" version instead:
+Push images using:
```

Comment:
I suggest also adding instructions on how to just generate the dockerfiles without building the images.

Reply:
Done!